.eslintrc.js

```diff
@@ -366,8 +369,12 @@ module.exports = {
         'jest/no-disabled-tests': 'warn',
         'jest/no-focused-tests': 'warn',
         'jest/no-identical-title': 'error',
+        'jest/no-mocks-import': 'off',
```

Comment:
pourquoi a-t-on besoin de la désactiver ?

Reply:
J'ai un problème avec 
`import { navigate } from '__mocks__/@react-navigation/native'`
-> Mocks should not be manually imported from a __mocks__ directory. Instead use `jest.mock` and import from the original module path

Il a raison, mais navigate n'existe pas dans @react-navigation/native : je peux créer un ticket pour investiguer ça